Doc/faq/general.rst

```diff
@@ -222,7 +222,7 @@ releases are announced on the comp.lang.python and comp.lang.python.announce
 newsgroups and on the Python home page at https://www.python.org/; an RSS feed of
 news is available.
 
-You can also access the development version of Python through Mercurial.  See
+You can also access the development version of Python through Git.  See
 https://docs.python.org/devguide/faq.html for details.
 
```

Comment:
Update:
https://docs.python.org/devguide has been updated to github.

At the moment /faq.html is still alive, but will be removed soon (see https://github.com/python/devguide/issues/114)

Let's remove the reference to faq.html here :)